app/models/weather.rb

```diff
@@ -22,6 +22,25 @@ def chance_of_rain(day_index)
     raw_day["precipProbability"] * 100
   end
 
+  def precip_type(day_index)
+    type = weather_info["daily"]["data"][day_index]["precipType"]
+    if type == nil
+      type = "precipitation"
+    end
+    type
+  end
+
+  def precip_icon(day_index)
+    type = weather_info["daily"]["data"][day_index]["precipType"]
+    icon = 'fas fa-cloud-rain'
```

Comment:
I'm a little worried about showing a rain cloud as the default precip type when there's no or very low chance of rain. Perhaps only show it if it's <10%